src/scripts/main.js

```diff
@@ -1,7 +1,50 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+
+  const div = document.createElement('div');
+
+  if (type === 'success') {
+    div.className = 'success';
+    div.style.backgroundColor = '#c0ddb6';
```

Comment:
```suggestion
    div.className = type;
```
Ти дублюєш рядок, який міститься в змінній 'type'. Можна просто один раз всередині функціії записати, що у div буде класс, назва якого міститься у змінній type, і не потрібно перевірок через switch case

Просто тричі виклич цю функцію з різними аргументами type

Також backgroundColor не потрібно задавати. Він вже є у класі